online.athn.browser.gnome.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id": "online.athn.browser.gnome",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "reference-browser-gnome",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=network"
+    ],
+    "build-options": {
+        "prepend-path": "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
```

Comment:
build must be offline, this is not allowed, and will fail.

Reply:
see for the tools that helps if you don't have a tarball with vendored crates.

https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo